### PR TITLE
Adds checkin service

### PR DESCRIPTION
**Pull request by gabibguedes**

### CODE REVIEW

File: src/pages/Checkin/index.tsx [hunk 1]
```diff
@@ -1,55 +1,77 @@
...
 export function Checkin(){
     const [local, setLocal] = useState('')
+    const [duration, setDuration] = useState(1)
     const [errorMessage, setErrorMessage] = useState('')
     const { user } = useAuth()
     const navigate = useNavigate()
+    const { createCheckin, isCreatingCheckin } = useCheckin()
 
-    const handleOnChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const localOnChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```
Comment (GabrielDVpereira) [inline src/pages/Checkin/index.tsx]: how about onLocalChange the name of this function?

File: src/pages/Checkin/index.tsx [hunk 2]
```diff
@@ -1,55 +1,77 @@
...
         setLocal(e.target.value)
         setErrorMessage('')
     }
 
-    const send = () => {
+    const durationOnChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```
Comment (GabrielDVpereira) [inline src/pages/Checkin/index.tsx]: how about "onDurationChange" as the name of this function?

**Review by GabrielDVpereira — APPROVED** [review]
💯 Just check out some suggestions I made

File: src/pages/Checkin/index.tsx (hunk 1 above)
Comment (gabibguedes) [reply src/pages/Checkin/index.tsx]: good sugestion

File: src/pages/Checkin/index.tsx (hunk 2 above)
Comment (gabibguedes) [reply src/pages/Checkin/index.tsx]: done

**Pull request merged**